addon/popup.js

```diff
@@ -73,8 +73,10 @@ function renderTabList(tabs, containerSelector, eventLabel) {
     let li = document.createElement("li");
     let image = document.createElement("span");
     let text = document.createElement("span");
+    let dismiss = document.createElement("span");
```

Comment:
I think you should use a button here, for accessibility, so it is possible to use the tab key to select this element and the space bar to press it.